lib/app/init/recipe_finder_app.dart

```diff
@@ -12,6 +13,7 @@ class RecipeFinderApp extends StatelessWidget {
     return ApplicationStarterWidget(
       routeManager: RouteManagerImpl(),
       app: (context) => MaterialApp(
+        builder: MainBuild.build,
```

Comment:
Creating a widget named 'ConnectivityWidget' to track the application's connectivity status and provide appropriate feedback to the user in case of connection interruptions, wrapping it with a builder, and designing it in an extensible structure can be a more appropriate approach.